pytest-wdl-tests/tests/whitespace/mixed.wdl

```diff
@@ -0,0 +1,31 @@
+# This workflow has 2-space indents and one tab at the heredoc! It is intended to fail miserably.
+
+version 1.0
+
+workflow mixed {
+  input {
+    String message = "I am a document. I am mixed. Let me fail!"
```

Comment:
Really? This looks fine to me?

If I'm reading this right: The engine will strip the common spacing from the start of the command section, and then the python heredoc will have a single tab indentation in it?

Reply:
> Failed to read task definition at line 17 column 6 (reason 1 of 1): Cannot mix leading whitespace characters in command: [" ", "\t"]
I get this error message, I believe it fails because of the tab in heredoc
I did also try a version where I put a tab on line 6 or 7 and that failed similarly